src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+public class Deadline extends Task{
+
+    protected String by;
```

Comment:
Name of variable can be more descriptive since this can be understood as the person who created the task or the actual deadline of the task